tests/admin/test_forms.py

```diff
@@ -123,29 +129,113 @@ def test_clean_existing_pending_link(self, existing_email):
         assert errors == {"email": [error_message]}
 
 
-class TestEnterpriseCustomerForm(unittest.TestCase):
+class TestEnterpriseCustomerAdminForm(unittest.TestCase):
     """
-    Tests for EnterpriseCustomerForm.
+    Tests for EnterpriseCustomerAdminForm.
     """
 
     def setUp(self):
         """
         Test set up.
         """
-        super(TestEnterpriseCustomerForm, self).setUp()
+        super(TestEnterpriseCustomerAdminForm, self).setUp()
         self.idp_choices_mock = mock.Mock()
         patcher = mock.patch("enterprise.admin.forms.utils.get_idp_choices", self.idp_choices_mock)
         patcher.start()
         self.addCleanup(patcher.stop)
+        self.catalog_id = random.uniform(1, 1000000)
```

Comment:
@haikuginger nit: `catalog_id` becomes float in this case. You might want to use `random.randint`, or better yet, FAKER.random_int.